app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -324,4 +337,40 @@ EnabledDynamicFieldForm.propTypes = {
   digitalObjectType: PropTypes.string.isRequired,
 };
 
+DynamicFieldCategory.propTypes = {
+  category: PropTypes.shape({
+    id: PropTypes.number,
+    displayLabel: PropTypes.string,
+    children: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.number,
+      }),
+    ),
+  }).isRequired,
+  enabledFieldDataCallback: PropTypes.func.isRequired,
+  disabled: PropTypes.bool.isRequired,
+};
+
+DynamicFieldGroup.propTypes = {
+  group: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    displayLabel: PropTypes.string.isRequired,
+    children: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.number,
+      }),
+    ),
+  }).isRequired,
+  enabledFieldDataCallback: PropTypes.func.isRequired,
+  disabled: PropTypes.bool.isRequired,
+};
+
+DynamicField.propTypes = {
+  field: PropTypes.shape({
+    id: PropTypes.number,
+  }).isRequired,
+  enabledFieldDataCallback: PropTypes.func.isRequired,
+  disabled: PropTypes.bool.isRequired,
+};
```

Comment:
Thank you for adding these propTypes!  I want to clean up our remaining eslint errors and add an eslint check to the CI task.  Things like this really help. 👍